### ISSUE TITLE
Proposal: Provide a QuickStart-Sample solution

### ISSUE THREAD

**Issue by glatzert** [SharpGenTools/SharpGenTools#130]
For people like me, it would be super helpful, if you could provide a minimal sample solution containing everything which is necessary to run a successful build.

We'd see how the output should look like and would probably be able to see differences to own our own usage and perhaps be able to deduct, what we are doing wrong.

**Comment by Kharzette** [comment]
Yes following the getting started guide just doesn't work.  The only alternative is looking at SharpDX which is enormously complicated.

I've been trying for hours just to get it to handle a simple cpp class with an int in it.
